### DISCUSSION TITLE
Consuming sales orders and invoices via the SAP cloud SDK

### DISCUSSION THREAD

**Discussion by adam91holt** [SAP/cloud-sdk-js#3836] [Q&A] [answered]
I've managed to get the SAP cloud SDK setup in TypeScript and have been able to successfully read customers (Business Partners) and products using the SDK.

I'm now wanting to pull out invoices via the APIs on offer. However, there are just so many it's very confusing to someone not too familiar with SAP. 

Can anyone point me to which endpoints I should be looking at and maybe an example of implementation?

Preference is to use the ODATAv4 Endpoints if possible!

Thank you!

**Comment by marikaner** [comment, marked as answer]
Hey @adam91holt, sorry for overlooking your question here. We from the SAP Cloud SDK are not experts on the APIs from SAP Business Accelerator Hub. I can only suggest to browse https://api.sap.com/ based on the information you have (maybe https://api.sap.com/allprocess can be helpful as well).
